tests/nightly/interfaces-openvswitch/task.yaml

```diff
@@ -16,19 +15,20 @@ details: |
     are carried through the socket, in this test we exercise bridge and port creation,
     list and deletion.
 
-# marking as manual due to errors during prepare
-manual: true
-
 prepare: |
     #shellcheck source=tests/lib/pkgdb.sh
     . "$TESTSLIB/pkgdb.sh"
-
+    
     echo "Given openvswitch is installed"
     distro_install_package --no-install-recommends openvswitch-switch
 
     # Ensure the openvswitch service is started which isn't the case by
     # default on all distributions
-    systemctl enable --now openvswitch
+    if find / -name openvswitch-switch.service | MATCH "openvswitch-switch.service"; then 
```

Comment:
Isn't this a bit brute-force? Can we use something that systemd offers to check if a service exists?

Reply:
I agree with zyga,`systemctl status..` should help here.